src/tests/networkcontrol/NetworkControl_Assign_001.test.js

```diff
@@ -0,0 +1,41 @@
+import {
+  pluginDeactivate,
+  pluginActivate,
+  assignNetworkAdapter,
+} from '../../commonMethods/commonFunctions'
+import constants from '../../commonMethods/constants'
+
+export default {
+  title: 'NetworkControl - Assign 001',
+  description: 'Check the Assign Functionality of Network Control Module',
+  steps: [
+    {
+      description: 'Check if NetworkControl Plugin is stopped correctly',
+      test: pluginDeactivate,
+      params: constants.networkControlPlugin,
+      assert: 'deactivated',
+    },
+    {
+      description: 'Check if NetworkControl Plugin is started correctly',
+      test: pluginActivate,
+      params: constants.networkControlPlugin,
+      assert: 'activated',
+    },
+    {
+      description: 'Invoke Assign',
+      sleep: 5,
+      test() {
+        return assignNetworkAdapter().call(this, 'eth0')
+      },
+      validate(res) {
+        //TODO - Update validation by confirming that assign happened
```

Comment:
How can we actually validate that the test has happened? Just be checking if the device is still online?
@pwielders @MFransen69  what does the "assign" do on the network adapter?

Reply:
Hi @wouterlucas ,

The tests were updated with TODO, as I don't know the validation for these and asked Divya how to validate assign/ flush / reload / request (Sent attachment in the Email with my queries). Once I get clarity on these I will update all of these.

Reply:
Lets park the assign tests for now so we can move the networkcontrol tests forward.